src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/ImportProcessingJob.cs

```diff
@@ -111,6 +112,12 @@ public async Task<string> ExecuteAsync(JobInfo jobInfo, IProgress<string> progre
                 {
                     throw;
                 }
+                catch (RequestFailedException ex) when (ex.Status == 403)
+                {
+                    _logger.LogInformation(ex, "Import processing operation is failed.");
```

Comment:
Could the message be changed to indicate that the failure was due to being unauthorized?

Reply:
Updated with unauthorized warning message.